docs/src/pages/index.page.tsx

```diff
@@ -133,7 +135,6 @@ const HomePage = ({ colorMode, setThemeOverride, themeOverride }) => {
       monoFont:
         '"Fira Mono", "DejaVu Sans Mono", Menlo, Consolas, "Liberation Mono", Monaco, "Lucida Console", monospace',
       fontSize: '16px',
-      lineHeight: '1.5',
```

Comment:
Remove it because of this error

<img width="825" alt="Screen Shot 2022-03-04 at 14 45 55" src="https://user-images.githubusercontent.com/11983489/156859809-6c629120-1d7c-445b-a9ac-c01711149269.png">

Reply:
Do we know why it's not present? Should we create a ticket to add it?

Reply:
`SandpackThemeProp` doesn't have it so I think we can't add it? 

Also @zchenwei  [said](https://github.com/aws-amplify/amplify-ui/pull/1483#r821022178) we are removing Sandpack https://github.com/aws-amplify/amplify-ui/pull/1395 I think It might not matter